modules/op-tkn-hub-interaction.adoc

```diff
@@ -0,0 +1,91 @@
+// Module included in the following assemblies:
+//
+// *  cli_reference/tkn_cli/op-tkn-reference.adoc
+
+[id="op-tkn-hub-interaction_{context}"]
+= Hub interaction commands
+
+Interact with Tekton Hub for resources such as tasks and pipelines.
+
+== hub
+Interact with hub.
+
+.Example: Display help
+[source,terminal]
+----
+$ tkn hub -h
+----
+
+.Example: Interact with hub API server
+[source,terminal]
+----
+$ tkn hub --api-server https://api.hub.tekton.dev
+----
+
+[NOTE]
+====
+For each command examples given below, you can get the corresponding sub-commands and flags by executing `tkn hub <command> --help`.
+====
+
+== hub downgrade
+Downgrade an installed resource
+
+.Example: Downgrade an installed resource
+[source,terminal]
+----
+$ tkn hub downgrade task -n mynamespace --to <resourceVersion>
+----
+
+== hub get
+Get resource manifest by its name, kind, catalog, and version.
+
+.Example: Get resource manifest
+[source,terminal]
+----
+$ tkn hub get [pipeline | task] --from tekton --version <resourceVersion>
+----
+
+== hub info
+Display information about a resource by its name, kind, catalog, and version.
+
+.Example: Display resource information
+[source,terminal]
+----
+$ tkn hub info task --from tekton --version <resourceVersion>
+----
+
+== hub install
+Install a resource from a catalog by its kind, name, and version.
+
+.Example: Install a resource
+[source,terminal]
+----
+$ tkn hub install task -n mynamespace --from tekton --version <resourceVersion>
```

Comment:
`$ tkn hub install task mytask  --from tekton`

Reply:
What happens when the user wants to install a specific version of the task from tekton hub? I guess if the version is omitted, then by default the latest version available will be installed, which might not always be desired.